index.js

```diff
@@ -21,33 +18,25 @@ fs.mkdir(pathName, { recursive: true }, (error) => {
 
 fs.readFile(file, 'utf8', (err, data) => {
     if (err) {
-        return console.log(err)
+        throw err;
     }
 
     // the data is split into a new array called clippings
     const clippings = data.split('==========');
 
     // declare an empty array that will hold the books & highlights
-    let books = [];
-
-    // searchBooks searches in 'books' array & returns index of book if it exists
-    function searchBooks(books, title) {
-        for(let i = 0; i < books.length; i++) {
-            if (books[i].title === title) {
-                bookIndex = i;
-                return;
-            }
-        }
-        bookIndex = -1;
-        return bookIndex;
-    }
+    const books = {};
 
     for(let i = 0; i < clippings.length; i++) {
 
         // split each clipping in clippings into component parts whenever there's a new line
-        let clipping = clippings[i].split('\r\n');
```

Comment:
not sure what the "\r" did/does, but it was causing problems on my machine. Might be a windows vs unix operating system thing. Don't know...